Doc/library/ast.rst

```diff
@@ -2155,7 +2155,7 @@ and classes for traversing abstract syntax trees:
    is no guarantee that the parse (or success of the parse) is the same as
    when run on the Python version corresponding to ``feature_version``.
 
-   If source contains a null character (``\0``), :exc:`ValueError` is raised.
+   If source contains a null character (``\0``), :exc:`SyntaxError` is raised.
```

Comment:
I feel we can remove this line, since there's no longer anything special here (a null byte is just like any other form of invalid syntax). The versionchanged note is enough.